src/webpack/loaders/style.js

```diff
@@ -32,17 +33,17 @@ export default {
     // importLoaders: use the following postcss-loader in @import statements
     // modules: enable css-modules
     const cssLoaders = [
-      `css?${options.cssModules ? 'modules' : ''}&sourceMap&importLoaders=1&localIdentName=${localIdentName}`,
+      `css?${options.cssModules ? 'modules' : ''}&${options.sourceMaps ? 'sourceMap' : ''}&importLoaders=1&localIdentName=${localIdentName}`,
```

Comment:
You could probably extract `${options.sourceMaps ? 'sourceMap' : ''}` to a variable.